app/assets/javascripts/components/common/popover.jsx

```diff
@@ -4,17 +4,23 @@ const Popover = React.createClass({
   displayName: 'Popover',
 
   propTypes: {
+    plus_right_aligned: React.PropTypes.bool,
     is_open: React.PropTypes.bool,
     edit_row: React.PropTypes.node,
     rows: React.PropTypes.node
   },
 
   render() {
     let divClass = 'pop';
+    divClass += ' right-aligned';
```

Comment:
One last bit of cleanup is possible now... may as well move this into the `let` in the line above.